### ISSUE TITLE
Merge servers in a multi-room server

### ISSUE THREAD

**Issue by JarnoVgr** [MrGreenGaming/MTA-Resources#38]
I believe @AfuSensi made a start on this. This really needs a higher priority to give the server more attention in the server browser list.

**Comment by AfuSensi** [comment]
I agree.
I will dedicate this month to it, so it should be done within. Some help would be greatly appreciated though.

**Comment by JarnoVgr** [comment]
How can we help you? You could put it up on a separate branch here, or on a branch on your local repo.

**Comment by EsDeKa** [comment]
I will help, I have some notes on how to do the maps so I could work on that for you

**Comment by JarnoVgr** [comment]
I assume the multi-room stuff would work with every kind of gamemode we create. So we could also do freeroam and such, right?

**Comment by AfuSensi** [comment]
Sorry for the late respons, been getting it ready for github, so that people actually know how i want to reach it and that it's readable for working together. 
I've based it on Bonsai multigamemode alot, so the script wrappers etc are already done.

I started a long time ago so i have to make it up to date too.

We can indeed add other gamemodes except race to it (i added "play" gamemode as a arena for instance), but the gamemode needs to be rewritten to fit its purpose ofcourse.

I also looked around to see if i can add a branch, but i need to look in to that.

Sorry for the wait, youll see more very soon.
